src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationValidBlockIndent.java

```diff
@@ -177,3 +177,13 @@ void create() { //indent:4 exp:4
         }; //indent:8 exp:8
     } //indent:4 exp:4
 } //indent:0 exp:0
+
+class OneLineCode { //indent:0 exp:0
+    void block() { //indent:4 exp:4
+        { if (true) return; }  //indent:8 exp:8
+    } //indent:4 exp:4
+    void tryCatchDoWhile(java.io.Closeable is) throws Exception { //indent:4 exp:4
+        if (is != null) { try { is.close(); } catch (Exception e) {} }  //indent:8 exp:8
+        if (is != null) { do { is.close(); } while (is != null); }  //indent:8 exp:8
```

Comment:
please always use test case from issue.
`Stream.of(false).map(x -> { if (x) return 0; else return 1; }); `

It is very good practice.
I do understand that now you well aware of Indentation logic so you can provide "better" test cases. BUT we need as much test cases as possible. And we need to make sure that is logic change all cases are working in the same way. In best scenario test inputs should be created as you do not know logic at all (blackbox).

Reply:
done.